server/services/interventionsService.ts

```diff
@@ -34,13 +33,14 @@ export interface ReferralFields {
 export interface DraftReferral extends WithNullableValues<ReferralFields> {
   id: string
   createdAt: string
+  serviceUser: ServiceUser
 }
 
 export interface SentReferral {
   id: string
-  createdAt: string
+  sentAt: string
```

Comment:
why has this changed?

Reply:
this is just a mistake in the expected response. the sent referral has a top level id, sentAt, referenceNumber, and then all the draft referral fields in the 'referral' object